docs/docs-content/integrations/nutanix-csi.md

```diff
@@ -0,0 +1,92 @@
+title: "Nutanix CSI"
+description: "Learn how to use the Nutanix CSI pack to provide persistent storage for your applications."
+hide_table_of_contents: true
+type: "integration"
+category: ["storage", "amd64"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: "https://registry.dev.spectrocloud.com/v1/nutanix-csi/blobs/sha256:7944cb5fecaaac0b5d5bf47a311ab80573147a34ac438954a70f97c69d65d733?type=image/png"
+tags: ["packs", "Nutanix", "Storage"]
+---
+
+The Nutanix Container Storage Interface (CSI) pack provides persistent storage for stateful applications. The pack consists of two charts - **nutanix-csi-storage** and **nutanix-cloud-provider**.
+
+The Nutanix Container Storage Interface (CSI) Volume Driver chart leverages Nutanix Volumes and Nutanix Files to provide scalable and persistent storage for applications.
+
+The Nutanix Cloud Provider chart is a plugin that allows the integration of the Nutanix Acropolis Hypervisor (AHV) platform with Kubernetes by implementing a node controller function.
+
+
+## Versions Supported
+
+- 2.6.6
+
+
+## Prerequisites
+
+- A Nutanix Prism Central account.
+
+- A Nutanix Prism Element cluster created.
+
+- A Nutanix cloud registered with Palette with the name `nutanix`. For more information, refer to [Register Nutanix Cloud](../clusters/data-center/nutanix/register-nutanix-cloud.md).
+
+- The Palette Kubernetes cluster must be version 1.20 or higher.
+
+
+## Parameters
+
+The table below lists commonly used parameters you can configure when adding the Nutanix CSI pack.
+
+|**Parameter**|**Description**|**Default**|
+|-------------|---------------|-----------|
+|`nutanix-cloud-provider.createConfig`| Creates a config for the Nutanix Cloud Provider. The existing config will be used if this parameter is set as `false`. |`true`|
+|`nutanix-cloud-provider.configName`| Name of the ConfigMap for the Nutanix Cloud Provider config. |`nutanix-config`|
+|`nutanix-cloud-provider.createSecret`| Creates a secret for the Nutanix Cloud Provider. The existing secret will be used if this parameter is set as `false`. |`true`|
+|`nutanix-cloud-provider.enableCustomLabeling`| Adds additional and custom Nutanix labels to nodes. |`false`|
+|`nutanix-cloud-provider.topologyDiscovery.type`| Defines how topology will be discovered (Prism or Categories). |`Prism`|
+|`nutanix-cloud-provider.podAnnotations`| Adds annotations to the Cloud Provider Pod. |`{}`|
+|`nutanix-csi-storage.volumeClass`| Activates the Nutanix Volumes Storage Class. |`false`|
+|`nutanix-csi-storage.volumeClassName`| Name of the Nutanix Volumes Storage Class. |`nutanix-volume`|
+|`nutanix-csi-storage.volumeClassRetention`| Retention policy for the Volumes Storage Class (Delete or Retain). |`Delete`|
+|`nutanix-csi-storage.fileClass`| Activates Nutanix Files Storage Class. |`false`|
+
+
+Review the [Nutanix CSI Pack](https://github.com/spectrocloud/pax/edit/master/stable/storage/nutanix-csi-2.6.6/README.md) GitHub page for a complete parameters list.
```

Comment:
I would move this to the end of the sentence above. That way, we reduce the chance of someone missing this when skimming.

Reply:
Totally agree!